src/common.speech/ConversationTranslator/ConversationConnection.ts

```diff
@@ -0,0 +1,460 @@
+import { IDisposable } from "../../common/Exports";
+import {
+    ConversationExpirationEventArgs,
+    ConversationParticipantsChangedEventArgs,
+    ConversationTranslationCanceledEventArgs,
+    ConversationTranslationEventArgs,
+    Participant,
+    ParticipantChangedReason } from "../../sdk/ConversationTranslator/Exports";
+import {
+    Connection,
+    ConnectionEventArgs,
+    PropertyId,
+    SessionEventArgs,
+    SpeechTranslationConfig} from "../../sdk/Exports";
+import {
+    ConversationReceivedTranslationEventArgs,
+    LockRoomEventArgs,
+    MuteAllEventArgs,
+    ParticipantAttributeEventArgs,
+    ParticipantEventArgs,
+    ParticipantsListEventArgs } from "./ConversationTranslatorEventArgs";
+import {
+    ConversationTranslatorCommandTypes,
+    ConversationTranslatorMessageTypes,
+    IConversationConnection,
+    IInternalConversation,
+    IInternalParticipant,
+    InternalParticipants} from "./ConversationTranslatorInterfaces";
+import { ConversationTranslatorRecognizer } from "./ConversationTranslatorRecognizer";
+
+/**
+ * Conversation client.
+ */
+// tslint:disable-next-line: max-classes-per-file
+export class ConversationConnection implements IConversationConnection, IDisposable {
+
+    /** websocket for outgoing commands and text messages and incoming messages */
+    private privConversationRecognizer: ConversationTranslatorRecognizer;
+    private privRecoConnection: Connection;
+    private privIsConnected: boolean = false;
```

Comment:
Are you using this to indicate that we are connected overall? If not, please rename this privRecoIsConnected for clarity

Reply:
This has been removed.